### ISSUE TITLE
Resolution > 16:9 / ultra-Widescreen support

### ISSUE THREAD

**Issue by Laserstrahl** [Albeoris/Memoria#186]
My Display has a resolution of 3840x2160, when I start the game with widescreensupport, the scene is only 4:3. 
When I start with 2560x1440 (windowed) everything is fine and 16:9. I was guessing that using my 21:9 resolution would have black bares left and right, and the displayed content would bei in 16:9... 
I can provide screenshots and config data if that helps.
I found people complaining about this bug on Steam and Reddit too.

Edit:
Basically this is not a bug, since the code does what it is ment for:

https://github.com/Albeoris/Memoria/blob/bb5ae2ce7aef96fa20d2849fbd24518647799304/Assembly-CSharp/Global/SFX/SFX.cs

check:  public static void UpdateScreenSize()

**Comment by Albeoris** [comment]
The simplest solution would be to limit the maximum screen width to 16:9 on ultra-wide monitors. 

It only makes sense to do full support for ultra-wide screens if we redraw all the backgrounds, since almost none of them is wide enough for such screens.

**Comment by Marlle** [comment]
Widescreen is not working at all on 2560*1080. If I choose 1920*1080 borderless it makes a nice 16:9 window, but on 2560*1080 it reverts back to 4:3 with black bars

**Comment by faospark** [comment]
the mod only support 16:9 aspect ratio.

**Comment by snouz** [comment]
The problem is also owning a 16/10 and larger screen, which I don't.
Can someone with one these types screens can at least confirm that widescreen works when you select a 16/9 resolution? Can you still play windowed or are you 100% locked into using 4/3?

**Comment by snouz** [comment]
After getting headaches trying to make a dynamic system that works with every resolution, I kinda de-scoped this, as I found it's much easier to work with a set of fixed ratios. So I'm sticking with 16:9 / 16:10 for now.
